### ISSUE TITLE
mono-tools-* is broken on Ubuntu based distros

### ISSUE THREAD

**Issue by johnnyasantoss** [mono/mono#16322]
## Steps to Reproduce

1. Try to install `mono-tools-*` on an Ubuntu based distro

```log
$ sudo apt install mono-tools-devel
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 mono-tools-devel : Depends: libmono-cecil-private-cil (< 6.0.0.314) but 6.0.0.319-0xamarin4+ubuntu1804b1 is to be installed
E: Unable to correct problems, you have held broken packages.
```

### Current Behavior

The current package has a broken dependency (I think that the problem might be that packages on Debian based distros have a suffix)

### Expected Behavior

Install without any problems

## On which platforms did you notice this

[ ] macOS
[x] Linux
[ ] Windows

**Version Used**:

```log
$ mono --version
Mono JIT compiler version 6.0.0.319 (tarball Mon Aug 12 23:46:26 UTC 2019)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       altstack
        Notifications: epoll
        Architecture:  amd64
        Disabled:      none
        Misc:          softdebug
        Interpreter:   yes
        LLVM:          yes(600)
        Suspend:       hybrid
        GC:            sgen (concurrent by default)
```
## Stacktrace

Doesn't apply

**Comment by johnnyasantoss** [comment]
Apparently this is not the first time :thinking:  #7285

**Comment by directhex** [comment]
`apt-cache policy mono-tools-devel`?

**Comment by directhex** [comment]
Looks like the fix was accidentally reverted in debian/4.2-3xamarin1.

Pushing a re-fixed build to the build system

**Comment by johnnyasantoss** [comment]
@directhex Thanks! :)
